sql-schema/200.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `devices` ADD `co` varchar(80) CHARACTER SET latin1 NULL DEFAULT '0';
```

Comment:
I didn't know that we could modify the databases  with the webui ?? great that but how we do this ?